website/src/components/avatar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+export default function Avatar({
+  src,
+  alt,
+  href,
+  title
+  }) {
+  return (
+    <div class="avatar">
+      <a
+        class="avatar__photo-link avatar__photo avatar__photo--lg"
+        href={href}
+      >
+        <img
+          alt={alt}
+          src={useBaseUrl(src)}
+        />
+      </a>
+      <div class="avatar__intro">
+        <a
+          href={useBaseUrl(src)}
```

Comment:
Le lien texte amène actuellement sur l'image, il faudrait qu'il pointe sur le site :
```suggestion
          href={href}
```

Reply:
😅 Oh la la, merci !

Reply:
Et il manque ça aussi: `target="_blank" rel="noopener noreferrer"`

Reply:
(si tu peux l'ajouter dans ton commit ?)

Reply:
Ça l'a ajouté dans le commit en modifiant le commentaire ?

Reply:
Oui c'est bon !